pyroSAR/drivers.py

```diff
@@ -93,7 +103,7 @@ def identify(scene):
     """
     if not os.path.exists(scene):
         raise OSError("No such file or directory: '{}'".format(scene))
-    
```

Comment:
please indent these empty lines so that they do not show up as changed. In PyCharm there is a setting to apply this during code auto-formatting.